src/Amadeus/Client/RequestOptions/Fare/MPFeeId.php

```diff
@@ -0,0 +1,48 @@
+<?php
+/**
+ * amadeus-ws-client
+ *
+ * Copyright 2015 Amadeus Benelux NV
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * @package Amadeus
+ * @license https://opensource.org/licenses/Apache-2.0 Apache 2.0
+ */
+
+namespace Amadeus\Client\RequestOptions\Fare;
+
+use Amadeus\Client\LoadParamsFromArray;
+
+/**
+ * MasterPricer Fee Id request options.
+ *
+ * @package Amadeus\Client\RequestOptions\Fare
+ * @author Dieter Devlieghere <dieter.devlieghere@benelux.amadeus.com>
+ */
+class MPFeeId extends LoadParamsFromArray
+{
+    /**
+     * Fee Type
```

Comment:
I'd like to have all possible Fee Types as constants "TYPE_*" here so users know what values are possible (and supported by Amadeus). 

This is analogous what I do in other places, for example [here](https://github.com/amabnl/amadeus-ws-client/blob/master/src/Amadeus/Client/RequestOptions/Fare/MasterPricer/FFCriteria.php#L35)

Here's the list of possible fee types: 

```
Fare_MasterPricerTravelBoardSearch/fareOptions/feeIdDescription/feeId/feeType
Attribute type identification, coded codesets
Value	Description
ARP	Arp number
FBA	Free Baggage Allowance
FFI	Fare Family Information
NPS	No PNR Split
PTRAM	Price To Reach amount type
RTC	Half round trip combination
SBF	Search by FBA
SORT	Sorting option (FEE, NOFEE)
TOKEN	Token
UPB	Upsell per bound
UPH	Homogenous upsell
```

As seen here: https://webservices.amadeus.com/extranet/structures/viewMessageStructure.do?id=5769&serviceVersionId=4507&isQuery=true